src/lib/evil/unposix/pthread.h

```diff
@@ -245,6 +245,12 @@ UNIMPLEMENTED inline int pthread_attr_init(void* a, ...)
     return 0;
 }
 
+UNIMPLEMENTED inline int pthread_attr_destroy(void* a, ...)
+{
+    #warning pthread_attr_destroy is not implemented.
+    return 0;
+}
+
```

Comment:
maybe check if #12 already implements this one?

Reply:
Well, even if it does, it is not merged yet.